src/XrdTls/XrdTlsSocket.cc

```diff
@@ -178,7 +178,8 @@ XrdTls::RC XrdTlsSocket::Accept(std::string *eWhy)
 
 // An accept may require several tries, so we do that here.
 //
-do{if ((rc = SSL_accept( pImpl->ssl )) > 0)
+do{ERR_clear_error();
```

Comment:
I would remove this. The server has never had an issue with Accept(). I suppose the overhead here is miniscule but I'm just trying to be consistent. Oddly, why wasn't the error queue cleared for connect()?

Reply:
Removed, and I'd overlooked SSL_connect(). Thanks!